tools/schematics/package-lock.json

```diff
@@ -0,0 +1,1907 @@
+{
```

Comment:
Can you please remove this file from the commit. 
I guess schematics should not be installed within the source of the tools folder.